### DISCUSSION TITLE
【求助】有没有人知道养塞斯的材料？想提前准备

### DISCUSSION THREAD

**Discussion by Shota-Kaedehara** [share121/inter-knot#3035] [其他]
![1f35b851069e96c507c53244d94947d53c97b6b9](https://github.com/user-attachments/assets/4066dd6b-59bf-4e77-8147-33457635bd17)
感觉塞斯好冷啊，同人图都找不到多少……
要开学了，不知道还能刷多少材料。打算提前给塞斯准备好，这样住校回来就可以直接开刷了。
你们有没有关于塞斯所需的培养材料的信息？

**Comment by Nie7bai** [comment]
![Uploading 赛斯.png…]()
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by Nie7bai** [reply]
照片好像加载不出来，悲
4-32-30（突破绿蓝红防护）（全突破）
25-75-250（技能绿蓝红电）（升到12）
核心技：9活态驱动（那个未知侵蚀体）60全坠如陨（蛮横力士）（点到F）

**Comment by I-m-Miss-Lucy-s-dog** [comment]
估計跟其他治安局角色一樣 不過核心技此素材估計是超雄電拳的的

**Comment by RaphaelHider** [comment]
找到两个相关信息，可以看看
![d34f170bf5c01feb680b1e38cdae5c1](https://github.com/user-attachments/assets/5cd8f279-0efa-433e-ba10-a95c3ee45344)
【最强异常辅助赛斯全面解析-哔哩哔哩】 https://b23.tv/Rn68HzN

**Comment by Shota-Kaedehara** [comment]
感谢各位绳匠/代理人
![NHhYY29MSlBzOUI5NHdKK0R3WndvbW5KY1d2Y1FqbWxUSG55K3RVUmdhOD0](https://github.com/user-attachments/assets/549c6b28-3287-4d65-83ed-157e2aae844c)
